src/tos/helpers/getTransComponents.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { ReactNode } from 'react';
+
+export default function getTransComponents(links: Array<string> | undefined) {
```

Comment:
You could get rid of the `if (links ...` by setting a default value of empty-array to links: `getTransComponents(links: Array<string> = []) {`

Reply:
Added default